examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -134,6 +140,27 @@ export default class App extends PureComponent {
     this._onSelectTileset(tileset);
   }
 
+  _getViewState() {
+    const {
+      viewState,
+      debugOptions: {showFrustumCullingMinimap}
+    } = this.state;
+    if (showFrustumCullingMinimap) {
+      return viewState;
+    }
+    return {main: viewState.main};
```

Comment:
Nit: I prefer compact code with less lines if it can be written cleanly
```suggestion
    const {viewState, debugOptions} = this.state;
    return (debugOptions.minimap) ? viewState :  {main: viewState.main};
```